src/main/java/com/darkona/adventurebackpack/item/ItemAdventure.java

```diff
@@ -14,12 +14,12 @@ public ItemAdventure() {
         super();
         setFull3D();
         setMaxStackSize(1);
+        setMaxDamage(1000);
```

Comment:
Yeah, it looks weird right?  It's the only way I could get Thaumcraft (or Witching Gadgets, I guess) to recognize the backpack as something that could be infused for the Translucency enchantment.  I don't actually understand how TC infusion recipes are set up, but I had a convo the other day about it (search the most recent "from:Kortako" in the Discord).

The intention is definitely not to change anything about the balance or values of the backpack, just to make it infusable, so if you know another way to fix it, I'd love to implement that instead.  1000 was also just a random number, it doesn't have to be that at all.  It literally just needs a damage value as far as I can tell.

Reply:
Ok. I found the root of the issue. According to the TC code (decompiled by IntelliJ), the enchantable item should be a tool to be allowed:
![image](https://github.com/user-attachments/assets/1bc96f59-9b6d-4dbf-bc7c-a53e3a43c0d3)

Default implementation of this method returns `true` when item is damageable. I suggest to override the method `isItemTool` instead of changing maximum damage. Look at implementation of `ItemBook` for example.

Reply:
Well, thank you very much for figuring that out for me, I didn't really know where to start.  That helps a lot.  I've implemented the override and tested it, and I'm much happier with the result.  See https://github.com/GTNewHorizons/AdventureBackpack2/pull/23/commits/cdbec328489a52444ba4ae65781e5d25a0673492